packages/jest/src/matcher.ts

```diff
@@ -47,6 +47,12 @@ export async function toBeAccessible(
     let a11yError: A11yError = new A11yError([], []);
     let receivedMsg = expectedMsg;
 
+    if (isTestUsingFakeTimer()) {
+        throw new Error(
+            'Cannot run accessibility check when fake timer is in use. Switch to real timer before invoking accessibility check.'
```

Comment:
Let's mention these are Jest fake timers and potentially link to the Jest doc. That'll help people debug failures with some specifics.

Reply:
Sounds good 👍 done  
Though the issue seems to occur also with third-party timer mock libs (e.g. @sinonjs/fake-timers) I guess Jest fake timers would be the majority use case.